lib/manager/npm/update/locked-dependency/common.ts

```diff
@@ -0,0 +1,28 @@
+export interface PackageLockDependency {
+  version: string;
+  resolved?: string;
+  integrity?: string;
+  dependencies?: Record<string, PackageLockDependency>;
+}
+
+export type PackageLockDependencies = Record<string, PackageLockDependency>;
+
+export interface PackageLockOrEntry {
+  lockfileVersion?: number;
+  version?: string;
+  dependencies?: PackageLockDependencies;
+  requires?: Record<string, string>;
+}
+
+export interface PackageJson {
```

Comment:
Don't we have this interface somewhere around?

Reply:
We should.. or it's in `type-fest` maybe